persistence/src/main/java/cz/muni/fi/pa165/dao/StoreDAO.java

```diff
@@ -15,4 +19,23 @@
      * @return all stores
      */
     List<Store> findAll();
+    
+    /**
+     * 
+     * @param name of store to be found
+     * @return Store
+     */
+    Store findByName(String name);
+    
+    @Query("select store.persons from Store store") 
+    List<Person> findAllPersonsInStore();
+
+    @Query("select p from Store store join store.persons p where p.id = ?1")
+    Person findPersonInStore(Long id_person);
+    
+    @Query("select store.bottles from Store store")
+    List<Bottle> findAllBottlesInStore();
```

Comment:
To iste, mne pride ze ti tam chyba nejaka referencia z ktoreho konkretne storu to vyhladavas